### PR TITLE
Added title labels to the camera and storage icon to easily make it e…

### PR DESCRIPTION
**Pull request by Breens-Mbaka**
1.This is an enhancement of the dialog picker, where I added titles to the respective camera icon and storage icon. I saw this will even make it easier for the user to know from which source they are picking the image from.This was done in the;
```
chooser_dialog.xml
```

2. I also reduced the icon size of the camera and storage icon and changed the alignment to better fit the titles together with the icons.

### CODE REVIEW

**Comment by ronnieotieno** [comment]
Screenshot 2022-01-22 at 07 28 31

Thanks, Breens, that's a nice contribution. 

Two things though:

1. What if you add titles under the camera/gallery images as opposed to beside them. The design should be horizontal with titles under them and not vertical (Just think its a better design  😊)

2. Please add a container containing the image plus the title and set onClick to it. Right now only the images are clickable but since the image and title are one thing they need to be clickable together. And the container should have tag ` android:background="?selectableItemBackground"` in order to provide feedback when clicked.

**Comment by Breens-Mbaka** [comment]
Okay I also agree having them horizontal is a good UX. What of the icon size should they remain the original size(before I changed them) of the width and height

**Comment by ronnieotieno** [comment]
Not a must, as long as the titles are under the images you can play with sizes and fonts.

File: build.gradle [hunk 1]
```diff
@@ -6,7 +6,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath "com.android.tools.build:gradle:4.2.2"
```
Comment (ronnieotieno) [inline build.gradle:9]: This out of the scope the commit, normally if you want to add somethings and not others in code you can use `git commit -up .` and enter `y` to add the change and `n` to skip the chnage.

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,6 +1,6 @@
 #Sun Jul 04 12:24:17 EAT 2021
 distributionBase=GRADLE_USER_HOME
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.7.1-bin.zip
```
Comment (ronnieotieno) [inline gradle/wrapper/gradle-wrapper.properties:3]: Here as well

File: imagepicker/build.gradle [hunk 3]
```diff
@@ -10,8 +10,6 @@ android {
     defaultConfig {
         minSdkVersion 16
         targetSdkVersion 31
-        versionCode 1
```
Comment (ronnieotieno) [inline imagepicker/build.gradle:13]: Why these changes? I know they don't matter though, any reason to remove them>?

Comment (Breens-Mbaka) [reply imagepicker/build.gradle:13]: For some reason when I updated my AS to Bumblebee this happened

File: build.gradle (hunk 1 above)
Comment (Breens-Mbaka) [reply build.gradle:9]: Sure

**Comment by ronnieotieno** [comment]
Nice, just make the changes and I am going to merge and do a release after.

**Comment by Breens-Mbaka** [comment]
Used the original IDE configurations according to the original version

**Review by ronnieotieno — APPROVED** [review]
